File: d1/t1.py
#!/usr/bin/env python3


count = 0
with open("input.txt") as infile:
    prev = int(infile.readline())
    current = infile.readline()
    while current != "":
        if int(current) > prev:
            count += 1
        prev = int(current)
        current = infile.readline()
print(count)

File: d3/t1.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    fstream infile("input.txt");
    while (infile) {
        string s;
        infile >> s;
        if (infile) {
            cout << s << endl;
        }
    }
    return 0;
}
